.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "src/llvm-project"]
 	path = src/llvm-project
 	url = https://github.com/llvm/llvm-project
-	branch = release/9.x
```

Comment:
Does this matter?  Can we just remove this line instead?  Isn't the sub module pinned to a certain git commit, regardless of the branch?    (Sorry I'm not a big user of submodule so I'm probably missing something here).

Reply:
This only ends up mattering for the behavior of `git pull` in a submodule, I believe? For the way we end up using the submodule, I don't think it matters much one way or the other. We can just leave it at whatever git set it to.

Reply:
I think that case maybe lets just drop this line, since otherwise the same thing will happen every 6 months when we forget to update it.  Also I imagine the updating between branches will more common than updating with a branch.. which lessens that value of having this line I think?

Reply:
Agreed then, lets drop the line.

Reply:
This matters if you want to run `git submodule update --remote`. As it is now, the update will set the branch "backwards" to release/9.x. It also makes the selection of the target more transparent. Right now it is at least a bit misleading. I was bitten by it when I tried to update LLVM as mentioned in https://github.com/WebAssembly/wasi-sdk/issues/133.

I can re-purpose the PR to delete the line if you prefer it though.

Reply:
Yes, I think deleting the misleading line would be better.

Reply:
up